tests/test_gbdesAstrometricFit.py

```diff
@@ -1058,6 +1061,114 @@ def test_inputCameraModel(self):
                     self.assertAlmostEqual(np.std(dDec), 0)
 
 
+class TestApparentMotion(lsst.utils.tests.TestCase):
+    """This class simulates an array of objects with random proper motions and
+    parallaxes and compares the results of
+    `lsst.drp.tasks.gbdesAstrometricFit.calculate_apparent_motion` against the
+    results calculated by astropy."""
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I don't see any real reason to use `setUpClass` here, why not just `setUp`? The main reason for `setUpClass` is for things that take a long time.

Reply:
Is there a downside to `setUpClass`? It just seems preferable to not rerun the setup if it's not necessary.

Reply:
`setUpClass` can cause confusion if a test modifies (often inadvertently!) a variable that was set, such that subsequent tests use the wrong value. When running in a multiprocessing environment (as we do with our pytest setup), `setUpClass` will run once per process, which could mean once per test. Unless you know that your setup is really slow, it's not worth the potential confusion of class variables potentially being modified in a test.